app/assets/javascripts/table_sorting.coffee

```diff
@@ -2,18 +2,19 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
-jQuery ->
-  $(document).on 'ready page:load', ->
-    # make the columns of all of the tables sortable
-    autosortable_tables = [ '#call_list table', '#completed_calls table', '#urgent_patients table', '#user-list table']
-    autosortable_tables.forEach ( table ) ->
-        stupidtable = $( table ).stupidtable()
+$(document).on 'turbolinks:load', ->
+  # make the columns of all of the tables sortable
+  sortTables()
 
-        # when a column is clicked, make sure to reset all of the other columns back to [-]
-        stupidtable.on 'beforetablesort', ( event, data ) ->
-            stupidtable.find( 'th' ).filter( ( i ) -> return i isnt data.column ).find( '.arrow' ).html( '[-]' )
-
-        # when a column is clicked, show an up or down arrow appropriately
-        stupidtable.on 'aftertablesort', ( event, data ) ->
-            arrow = if data.direction is $.fn.stupidtable.dir.ASC then '[&uarr;]' else '[&darr;]'
-            stupidtable.find( 'th' ).eq( data.column ).find( '.arrow' ).html( arrow )
+sortTables = ->
```

Comment:
Fun fact I learned after poking at coffeescript recently: You can export functions by slapping an `@` in front of em. Kinda nice! (This is good as is though.)